docs/NativeComponentsIOS.md

```diff
@@ -56,31 +56,42 @@ Then you just need a little bit of JavaScript to make this a usable React compon
 
 import { requireNativeComponent } from 'react-native';
 
-// requireNativeComponent automatically resolves this to "RNTMapManager"
+// requireNativeComponent automatically resolves 'RNTMap' to 'RNTMapManager'
 module.exports = requireNativeComponent('RNTMap', null);
+
+// MyApp.js
+
+render() {
+  return <MapView style={{ flex: 1 }} />;
+} 
 ```
 
+Make sure to use `RNTMap` here. We want to require the manager here, which will expose the view of our manager for use in Javascript. 
```

Comment:
I thought this could use some more clarification. 

https://github.com/facebook/react-native/issues/12853 
https://github.com/facebook/react-native/issues/3575